Sources/SwiftGenStringsCore/SwiftTokenizer.swift

```diff
@@ -1,8 +1,8 @@
 import Foundation
 
-class SwiftTokenizer {
+public struct SwiftTokenizer {
 
-    func tokenizeSwiftString(_ string: String) -> [SwiftLanguageToken] {
+    public static func tokenizeSwiftString(_ string: String) -> [SwiftLanguageToken] {
```

Comment:
Is there any reason for changing the behavior from instance method to a class method?
Instance methods are much easier to mock / replace as we can replace the instance. Not the same case with a class.

Reply:
No real reason, just thought that the call-site looks nicer that way. But you're right, that's a little more important